server/services/temporaryAbsencesService.ts

```diff
@@ -13,4 +13,14 @@ export default class TemporaryAbsencesService {
     const temporaryAbsences = await this.welcomeClientFactory(token).getTemporaryAbsences(agencyId)
     return temporaryAbsences.sort(compareByFullName)
   }
+
+  public async getTemporaryAbsence(agencyId: string, prisonNumber: string): Promise<TemporaryAbsence> {
+    const token = await this.hmppsAuthClient.getSystemClientToken()
+    const temporaryAbsence = await this.welcomeClientFactory(token).getTemporaryAbsence(agencyId, prisonNumber)
+    return {
+      ...temporaryAbsence,
+      firstName: convertToTitleCase(temporaryAbsence.firstName),
```

Comment:
formatting twice?

Reply:
Agree it isn't great - seemed less code than formatting in both the service get methods but I guess I could have extracted a private formatting method there... will sort in the backend instead as suggested - thank you

Reply:
Removed on latest commit